src/components/Card/Card.jsx

```diff
@@ -6,21 +6,21 @@ import { commerce } from '../../lib/commerce';
 
 
 
-const Card = ({products}) => {
+const Card = ({item}) => {
```

Comment:
There's some mixup between products and item in various places - products is a list of all the items, item is one item